### ISSUE TITLE
Is the MSE loss  in the code right?

### ISSUE THREAD

**Issue by sddi** [ChenglongChen/tensorflow-DeepFM#6]
https://github.com/ChenglongChen/tensorflow-DeepFM/blob/b0391f009c364c132ac927d39652519a473e7271/DeepFM.py#L129

In the condition that shows loss_type=="mse", use "self.out" directly or  adding "self.out = tf.nn.sigmoid(self.out)" and then use the "self.out"?

**Comment by ChenglongChen** [comment]
Sigmoid gives probability in the range (0, 1). For mse, you are safe to use self.out directly. If in any case, your target of the regression task lies in the range (0, 1), you can use sigmoid to squash self.out and then apply mse on that.
